content/course-structure.md

```diff
@@ -0,0 +1,47 @@
+# Course Structure
+
+Our courses are followed by a well-organized framework that combines interactive lessons, hands-on projects, and personalized support. It doesn't matter if you are a beginner or already have some knowledge in the field. We assure you that we will make it as simple and entertaining as possible.
+How will we do that?? You may be wondering... Well, don't worry! In this article, we will briefly explain how our system works.
+
+Whether you choose one of our bootcamps or short courses, we have worked based on our ***[special formula](https://4geeksacademy.notion.site/4geeksacademy/Mastering-Technical-Knowledge-984d2df394c44aedb05987311ccfcf06)*** in order to guarantee an exponential learning curve 📈. You will see that it will be entertaining, and you will learn things all the time without even realizing it!
+
+So, let's show you what you will find along this journey 😎
+
+## Basic structure
+
+The first thing that you need to know is that our courses are divided into modules, making it super easy for you to navigate through the material. Once you're all set and click the "start course" button, you'll be away to the next screen:
+
+![Course Structure](../images/course.structure.png)
+
+Up at the top of your screen, you'll find some really helpful information. It'll show you the percentage of progress you've made in the program, how many tasks you've already completed, and how many more exciting tasks you have ahead. It's like your personal progress tracker!
```

Comment:
This is extra: `you'll find some really helpful information.` remove it and go straight to what there is